packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -20,13 +20,14 @@ import priceSVG from "../../../../assets/icons/price.svg";
 import videoSVG from "../../../../assets/icons/video.svg";
```

Comment:
Maybe you can split the file to don't have 400 lines in a file ?

Reply:
It's a good idea, but I don't want to do this on this PR since it didn't modify `NewsFeedInput.tsx`.
This file is a little simplified in this PR : https://github.com/TERITORI/teritori-dapp/pull/809 (Just the post creation)